tests/test_utils.py

```diff
@@ -313,3 +313,16 @@ def test_check_status_code_for_missing_status_code(capsys, repo_url):
 
     captured = capsys.readouterr()
     assert captured.out == "NOTE: Try --verbose to see response content.\n"
+
+
+@pytest.mark.parametrize(
+    ("num_bytes, expected"),
+    [(3704, "3.6 KB"), (1153433, "1.1 MB"), (21412841, "20.4 MB")],
+)
+def test_get_file_size(num_bytes, expected, monkeypatch):
```

Comment:
Let's use a more explicit variable name:

```suggestion
def test_get_file_size(size_in_bytes, formatted_size, monkeypatch):
```